samples/simple/src/iwasm_main.c

```diff
@@ -354,6 +354,25 @@ static host_interface interface = { .send = uart_send, .destroy = uart_destroy }
 
 #endif
 
+
+
+static attr_container_t * read_test_sensor(void * sensor)
+{
+    //luc: for test
+    attr_container_t *attr_obj = attr_container_create("read test sensor data");
+    if (attr_obj) {
+        attr_container_set_string(&attr_obj, "name", "read test sensor");
```

Comment:
Should check return value, the original code doesn't check it, it may cause memory leak.

Reply:
I have modified the code to add the return value check.